packages/attention/src/component.tsx

```diff
@@ -137,10 +143,10 @@ const Arrow = forwardRef<HTMLDivElement, ArrowProps>((props, ref) => {
       className={arrowClasses}
       style={{
         // TW doesn't let us specify exactly one corner, only whole sides
-        borderTopLeftRadius: '4px',
+        borderTopLeftRadius: "4px",
         zIndex: 1,
         // border alignment is off by a fraction of a pixel, this fixes it
-        [`margin${arrowDirectionClassname(arrowDirection)}`]: '-0.5px',
+        [`margin${arrowDirectionClassname(arrowDirection)}`]: "-0.5px",
```

Comment:
Did we change the prettier config or something?

Reply:
Could be from my prettier-extension! Do we have a prettier config?